packages/flac/flac.0.1.2/opam

```diff
@@ -3,15 +3,18 @@ maintainer: "Romain Beauxis <toots@rastageeks.org>"
 authors: "The Savonet Team <savonet-users@lists.sourceforge.net>"
 homepage: "https://github.com/savonet/ocaml-flac"
 build: [
-  ["./configure" "--prefix" prefix] { os != "darwin" }
-  ["./configure" "CFLAGS=-I/usr/local/include" "LDFLAGS=-L/usr/local/lib" "OCAMLFLAGS=-ccopt -I/usr/local/include -cclib -L/usr/local/lib" "--prefix" prefix] { os = "darwin" }
+  ["./configure" "--prefix" prefix opamconfig:cflags opamconfig:ldflags opamconfig:ocamlflags]
   [make]
 ]
 install: [
   [make "install"]
 ]
 remove: ["ocamlfind" "remove" "flac"]
-depends: ["ocamlfind" "ogg"]
+depends: [
+  "opamconfig"
```

Comment:
Yes, it should, I think. The code does not depend on what is detected by `opamconfig`, so there is no reason to recompile when `opamconfig` is updated.